### PR TITLE
PullRequest for Issue87a: refine the generic debug level code

### PR DESCRIPTION
**Pull request by XuetaoLiu-S**
https://github.com/acquaman/AcquamanDataServer/issues/87

This is a second commit for Issue87, which provided the debugging level and centralized the usage of AMDSErrorMon. In this way, we don't need to worry about the configured debugging level when we are going to print out the message.

This changes is based on Branch Issue33, and should be merged to that branch

### CODE REVIEW

File: source/AMDSMainSGMScaler.cpp [hunk 1]
```diff
@@ -16,7 +16,7 @@ int main(int argc, char *argv[])
 
 	bool startServer = AMDSMain_Common::parseArgument("AcquamanDataServerSGMScaler", args, &interfaceType, &port);
 	if (startServer) {
-		AMDSRunTimeSupport::setDebugLevel(1);
+		AMDSRunTimeSupport::setDebugLevel(AMDSRunTimeSupport::All);
```
Comment (dretrex) [inline source/AMDSMainSGMScaler.cpp]: @davidChevrier, is your level 1 the same as `All` here?

**Comment by dretrex** [comment]
Time Estimate: 5 minutes

File: source/AMDSMainSGMScaler.cpp (hunk 1 above)
Comment (davidChevrier) [inline source/AMDSMainSGMScaler.cpp]: Nope. I want only the errors as the standard value.

Comment (dretrex) [inline source/AMDSMainSGMScaler.cpp]: Okay.  @XuetaoLiu-CLS can you change both of these levels to error only?  Once you do that you should be able to merge it in.

**Pull request merged**